docs/appendices/release-notes/unreleased.rst

```diff
@@ -221,5 +221,9 @@ Performance improvements
 Fixes
 =====
 
+- Fixed an issue that caused a ``NullPointerException`` if the :ref:`ANALYZE
+  <analyze>` statement was executed on tables with the text array type columns
```

Comment:
```suggestion
  <analyze>` statement was executed on tables with text array type columns
```

Reply:
Did this only apply to `text` ?

Reply:
actually it applies to all primitive types